packages/common/src/hooks/useIsSmallScreen.tsx

```diff
@@ -0,0 +1,6 @@
+import { useMediaQuery, useTheme } from '@material-ui/core';
+
+export const useIsSmallScreen = (): boolean => {
+  const theme = useTheme();
+  return useMediaQuery(theme.breakpoints.down('lg'));
```

Comment:
we'll need a few more options sorry. The viewports page in zeplin has these values as defaults:
- `> 1536 wide`, the detail panel visible
- `< 1536 wide`, the detail panel hidden
- `> 1440 wide` app drawer expanded
- `< 1440 wide` app drawer collapsed
- `> 1024 wide` app bar RH button label visible
- `< 1024 wide` condensed milk

whaddya say that we continue this specific pattern that you've used, and just add in `useIsLargeScreen` and `useIsMediumScreen`? would that be too restrictive?

Reply:
So I went a bit OTT and added `useXL`, `useLarge`, `useMed` : e99baa3

This was because I changed the breakpoints, and this was aligning somewhat with that. Additionally in hindsight it's actually probably only a matter of time before we support mobiles, in at least some way?

I am not 100% about the hooks and using `down` from the previous breakpoint and if that is just confusing. Open to changing them again if you find it confusing

Reply:
Oh I also added stories for the hooks because 1) they were helping me visualise and 2) actually stories for hooks would be useful (as if I haven't added enough things for us to do 🙄 )

Reply:
you've done heaps! thanks - I think the hook approach is much clearer than checking breakpoint levels and media sizes. that level of granularity is available if we really need it, I think that mostly we will only use small, medium etc. The use of hooks is consistent with other usages too.

Will update the RH drawer when this PR is merged in, and can fix up the tests to add in a test for smaller screen size too now :)